taggit/managers.py

```diff
@@ -309,13 +339,18 @@ def __lt__(self, other):
         return False
 
     def post_through_setup(self, cls):
-        self.related = RelatedObject(cls, self.model, self)
+        if RelatedObject is not None:  # Django < 1.8
```

Comment:
thanks @kaedroho, you were right it was leftover files from not doing a clean install of django. I agree with @collinanderson I'm sure I won't be the only person to not think to clear out the old django before upgrading.